Added notes related to security pre-notify list requests (#12941)
django/django: docs/internals/security.txt

   demonstrated ability to responsibly receive, keep confidential and
   act on these notifications.

.. admonition:: Security audit and scanning entities

>     As a policy, we do not add these types of entities to the notification
>     list. We are unable to approve all requests and it is not fair to give
>     some a competitive advantage.

Reviewer: I think I'd just have the first sentence:

```suggestion
    As a policy, we do not add these types of entities to the notification
    list.
```

receiving security notifications is a privilege granted at the sole
discretion of the Django development team, and that this privilege can
be revoked at any time, with or without explanation.

.. admonition:: Provide all required information

>     A failure to provide the required information in your initial contact
>     will count against you when making the decision on whether or not to
>     approve your request.

Reviewer: Does it actually count negative? (Maybe it does... 🙂)

```suggestion
    We cannot process you request unless you provide all the required information. 
```

I don't know. 🤷

Author: Based upon direct comments from other team members when deciding and my personal views, I'd say it does hurt their chances of approval. It's a bit of a challenge to trust a person with confidential information after they've shown they don't read/follow instructions. 

With that said, I'm not opposed to toning down the wording.

Reviewer: OK, let's keep it. Thanks!
